packages/core/src/i18n/arrayTranslations.ts

```diff
@@ -0,0 +1,40 @@
+export interface DefaultTranslation {
+    key: ArrayTranslationEnum,
+    default: (variable?: string) => string
+}
+
+export enum ArrayTranslationEnum{
+    addTooltip = 'addTooltip',
+    addAriaLabel = 'addAriaLabel',
+    removeTooltip = 'removeTooltip',
+    upAriaLabel = 'upAriaLabel',
+    downAriaLabel = 'downAriaLabel',
+    noSelection = 'noSelection',
+    removeAriaLabel = 'removeAriaLabel',
+    noDataMessage = 'noDataMessage',
+    deleteDialogHeader = 'deleteDialogHeader',
```

Comment:
I think title more precisely describes what this is for.
```suggestion
    deleteDialogTitle = 'deleteDialogTitle',
```